cli/src/infragpt/llm/providers/anthropic_provider.py

```
@@ -0,0 +1,211 @@
+import json
+from typing import List, Dict, Any, Iterator, Optional
+from anthropic import Anthropic
+import anthropic
+
+from ..base import BaseLLMProvider
+from ..models import StreamChunk, ToolCall
+from ..exceptions import (
+    AuthenticationError,
+    RateLimitError,
+    APIError,
+    ToolCallError,
```

Ruff (0.12.2)

`..exceptions.ToolCallError` imported but unused

Remove unused import: `..exceptions.ToolCallError`

(F401)